internal/expr/parser.go

```diff
@@ -624,7 +639,9 @@ func parseList[T any](p *Parser, parseFn func(p *Parser) (T, bool, error)) ([]T,
 
 		nextItem = p.skipByte(',')
 	}
-	return nil, false, errorAt(fmt.Errorf("missing closing parentheses"), openingParenLine, openingParenCol, p.input)
+	err := errorAt(fmt.Errorf("missing closing parentheses"), openingParenLine, openingParenCol, p.input)
```

Comment:
I see that openingParenLine and openingParenCol are redundant given that we have `cp`. We could remove them in this PR :P

Reply:
Removed.